internal/util/vault_tokens.go

```diff
@@ -225,3 +295,18 @@ func getToken(tenant, tokenName string) (string, error) {
 
 	return string(token), nil
 }
+
+func getCertificate(tenant, secretName, key string) (string, error) {
+	c := NewK8sClient()
+	secret, err := c.CoreV1().Secrets(tenant).Get(context.TODO(), secretName, metav1.GetOptions{})
+	if err != nil {
```

Comment:
if there is a global configuration, but the certificates are not in the tenants namespace but in the global one, this will fail?